docs/integrations/dell.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
Is this document path still valid ?  I saw the current docs directory don't have any integration now ?  would you mind to update this PR ?

Reply:
The document is moved and fixed the header~